examples/Python/python-django-webapp-sample/README.md

```diff
@@ -21,7 +21,6 @@ az appconfig create --name <config-store-name> --resource-group <resource-group-
 ```Powershell
 az appconfig kv set --name <config-store-name> --key testapp_settings_message --value "Hello from Azure App Configuration"
 az appconfig kv set --name <config-store-name> --key testapp_settings_font_size --value "30px"
-az appconfig kv set --name <config-store-name> --key testapp_settings_color --value "yellow"
```

Comment:
Nothing wrong with it. Just be accessibility-aware if using it for demos. For example, our Azure portal is accessibility compliant, but it doesn't mean it will be colorless.

Reply:
These are usually the first thing we bring up to demo. Also the yellow was a rather bright one… The text should be good enough.

Reply:
We have other samples that use a color background too. It's good to keep them consistent. We can change the value to something less bright if preferred, for example, azure :)

Reply:
Added back as `Azure`